master/reference/calicoctl/resources/networkset.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Network Set Resource (networkSet)
+---
+
+A network set resource (networkSet) represents an arbitrary set of IP subnetworks/CIDRs, allowing it
+to be matched by Calico policy.  Network sets are useful for applying policy to traffic
+coming from (or going to) external, non-Calico, networks.
+
+Each network set includes a set of labels, which are in the same "namespace" as those attached to
```

Comment:
We should probably avoid saying "namespace" here since it will likely confuse k8s users.